README.md

```diff
@@ -14,6 +14,22 @@ npm run build
 
 ## Deployment
 
+### Automated Deployment
+
+We now have a series of scripts that will help with automating the release process. To use these scripts, do the following.
+
+1. Run `npm install` in the project root to make sure all dependencies are installed at the project level and in the `./scripts` folder.
```

Comment:
Should we run `nvm use` first?

Reply:
That's a good point. I'm used to the setup where `nvm` automatically switches versions by looking at the `.nvmrc` but probably not everyone has that configuration.

I'll make the update.